src/functional/resources/repo-config.csv

```diff
@@ -1,6 +1,6 @@
-Repository's Location,Branch,Ignore Glob List,Ignore standalone config
-https://github.com/reposense/testrepo-Alpha.git,master,,
-https://github.com/reposense/testrepo-Beta.git,master,docs**,yes
-https://github.com/reposense/testrepo-Charlie.git,master,,
-https://github.com/reposense/testrepo-Delta.git,master,,
-https://github.com/reposense/testrepo-Delta.git,nonExistentBranch,,
+Repository's Location,Branch,Ignore Glob List,Ignore standalone config,Ignore Commits List
+https://github.com/reposense/testrepo-Alpha.git,master,,,
```

Comment:
a bit hard to visually verify the results... can we add hashes to make fakeAuthor empty in both functional n netlify?